input/fsh/DHPSickLeaveCarePlan.fsh

```diff
@@ -0,0 +1,71 @@
+Profile: DHPSickLeaveCarePlan
+Parent: CarePlan
+Id: dhp-sick-leave-careplan
+Title: "DHP Sick Leave CarePlan"
+Description: "Profile for documenting sick leave status in the healthcare system of Uzbekistan for legal purposes. This CarePlan is not intended for clinical management but serves to formalize the process of managing sick leave for patients, ensuring compliance with national regulations and facilitating communication between healthcare providers, employers, and relevant authorities."
+* ^experimental = true
+* ^status = #draft
+
+* category MS
+* category ^slicing.discriminator.type = #value
+* category ^slicing.discriminator.path = "coding.system"
+* category ^slicing.rules = #open
+* category contains sickLeave 1..1 MS
+* category[sickLeave] = SickLeaveCategoryCS#mserv-0005-00001
+* category[sickLeave] ^short = "Specifies that the CarePlan is for managing sick leave status"
+
+* subject MS
+* subject only Reference(UZCorePatient)
+* subject ^short = "For whom is the Certificate of Incapacity for Work opened"
+
+* addresses ^slicing.discriminator.type = #value
+* addresses ^slicing.discriminator.path = "concept"
+* addresses ^slicing.rules = #open
+
+* addresses contains reason 0..1 and diagnosis 0..1 MS
+* extension contains CarePlanStatusHistory named statusHistory 0..*
+
+* addresses[reason] from CarePlanReasonVS (required)
+  * ^short = "Reason for the sick leave"
+
+* addresses[diagnosis] from $icd-10-vs (required)
+  * ^short = "ICD-10 diagnosis"
+
+* status MS
+  * ^short = "Status: Sick Leave"
+  * ^binding.strength = #required
+* status from CarePlanStatusVS
```

Comment:
Some of the status will be from CarePlanStatusVS, but because we cannot add new codes, the extra codes will need to be in a new `extended-status`.